src/constants/index.js

```diff
@@ -5,7 +5,8 @@ import FeatureFlags from './featureflags';
 const Categories = Object.freeze({
   EVENT: { label: 'evento', color: 'dark-blue' },
   WORKSHOP: { label: 'workshop', color: 'dark-blue' },
-  MEDIA: { label: 'media', color: 'dark-blue' }
+  MEDIA: { label: 'media', color: 'dark-blue' },
+  ESCOLA: { label: 'media', color: 'dark-blue' }
```

Comment:
assim que confirmar com a equipa, qual o termo adequado, também terás de modificar aqui no `label`

Reply:
Podes mudar para Educacao @msantosdevlab por favor?
```suggestion
  EDUCACAO: { label: 'media', color: 'dark-blue' }
```